contracts/vendor/starkware-libs/starkgate-contracts-solidity-v0.8/contracts/starkware/starknet/solidity/IStarknetMessaging.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0.
+pragma solidity >0.6.0 <0.9.0;
+
+import "./IStarknetMessagingEvents.sol";
+
+interface IStarknetMessaging is IStarknetMessagingEvents {
+    /**
```

Comment:
let's use Natspec comments for Solidity contracts https://docs.soliditylang.org/en/v0.8.15/natspec-format.html#:~:text=Solidity%20contracts%20can%20use%20a,Language%20Specification%20Format%20(NatSpec).&text=NatSpec%20was%20inspired%20by%20Doxygen.

Reply:
This is a vendor contract, a duplicate from source. We are not looking to make any changes here, unless completely necessary.

Reply:
ignore as this is from the vendor file